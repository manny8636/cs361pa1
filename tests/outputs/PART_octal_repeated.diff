--- outputs/PART_octal_repeated.txt	2022-02-13 13:37:44.979532683 -0500
+++ expected/PART_octal_repeated.txt	2021-01-25 15:27:24.000000000 -0500
@@ -1 +1,2 @@
+INTEGER: 93
 Success!
