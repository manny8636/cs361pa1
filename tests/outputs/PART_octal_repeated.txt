Parsing ../data/octal_repeated.txt failed
