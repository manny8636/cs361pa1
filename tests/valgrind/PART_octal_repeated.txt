==372082== Memcheck, a memory error detector
==372082== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==372082== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==372082== Command: ../json -i ../data/octal_repeated.txt
==372082== 
==372082== 
==372082== HEAP SUMMARY:
==372082==     in use at exit: 2,064 bytes in 1 blocks
==372082==   total heap usage: 4 allocs, 3 frees, 10,848 bytes allocated
==372082== 
==372082== LEAK SUMMARY:
==372082==    definitely lost: 2,064 bytes in 1 blocks
==372082==    indirectly lost: 0 bytes in 0 blocks
==372082==      possibly lost: 0 bytes in 0 blocks
==372082==    still reachable: 0 bytes in 0 blocks
==372082==         suppressed: 0 bytes in 0 blocks
==372082== Rerun with --leak-check=full to see details of leaked memory
==372082== 
==372082== For lists of detected and suppressed errors, rerun with: -s
==372082== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
