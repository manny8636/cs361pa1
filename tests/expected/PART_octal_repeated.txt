INTEGER: 93
Success!
